### PR TITLE
Partial

### PR DESCRIPTION
**Pull request by cassiemak**
Thanks everyone's hard work
Partial now finally works with masonry. 
Changes
Partial:
1. Header
2. Pin-index =>  where masonry lives
3. User-show 

HTML:
1. Home => where it renders ----> Header, Pin-Index
2. User => where it renders ------> Header, User-show, Pin-Index

Toy Story = Story of my life! I miss my teddy bear.... 

Please review!!! Thanks!

### CODE REVIEW

File: app/assets/templates/user.html [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<div class="container-fluid background text-center">
+  <top-header></top-header>
+  <user-details></user-details>
+  
+  <div>
+    <index-Body></index-Body>
```
Comment (h8rry) [inline app/assets/templates/user.html]: should just use `<index-body>` as Angular's convention

File: app/assets/javascripts/angular/controller/user-controller.js [hunk 2]
```diff
@@ -9,8 +9,14 @@ app.controller('UserCtrl', ['$scope', '$http',
   // })
   
     $http.get('pins.json').success(function(data){
```
Comment (h8rry) [inline app/assets/javascripts/angular/controller/user-controller.js]: Indentation

File: app/assets/javascripts/angular/controller/user-controller.js [hunk 3]
```diff
@@ -9,8 +9,14 @@ app.controller('UserCtrl', ['$scope', '$http',
...
-    console.log('sucess on getting pins');
+    console.log('success on getting pins from UserCtrl');
     $scope.pins = data.pins;
   })
 
+    $http.get('/categories.json').success(function(data){
```
Comment (h8rry) [inline app/assets/javascripts/angular/controller/user-controller.js]: Indentation

File: app/assets/javascripts/angular/controller/home-controller.js.erb [hunk 4]
```diff
@@ -6,14 +6,18 @@ app.controller('HomeCtrl', ['$scope', '$http', function($scope, $http){
...
     // console.log(data);
     $scope.pins = data.pins;
     var numberOfPins = data.pins.length;
-    console.log(numberOfPins);
-    console.log(data.pins[numberOfPins-1]);
-    console.log('from controller I see', angular.element('.box').length,'boxes');
+    // console.log('from controller I see', angular.element('.box').length,'boxes');
   });
 
+  $http.get('/categories.json').success(function(data){
```
Comment (h8rry) [inline app/assets/javascripts/angular/controller/home-controller.js.erb:11]: Indentation

File: app/assets/templates/user.html (hunk 1 above)
Comment (cassiemak) [inline app/assets/templates/user.html]: OK. will amend

File: app/assets/javascripts/angular/controller/user-controller.js (hunk 2 above)
Comment (cassiemak) [inline app/assets/javascripts/angular/controller/user-controller.js]: OK. will amend

File: app/assets/javascripts/angular/controller/user-controller.js (hunk 3 above)
Comment (cassiemak) [inline app/assets/javascripts/angular/controller/user-controller.js]: k

File: app/assets/javascripts/angular/controller/home-controller.js.erb (hunk 4 above)
Comment (cassiemak) [inline app/assets/javascripts/angular/controller/home-controller.js.erb:11]: k

File: app/assets/javascripts/angular/directives/user-details.js.erb [hunk 5]
```diff
@@ -0,0 +1,20 @@
+app.directive('userDetails', ['$http', function($http){
+  // Runs during compile
+  return {
+    // name: '',
```
Comment (h8rry) [inline app/assets/javascripts/angular/directives/user-details.js.erb]: Let's remove all the comments

Comment (cassiemak) [inline app/assets/javascripts/angular/directives/user-details.js.erb]: ok

File: app/assets/javascripts/angular/controller/home-controller.js.erb [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+    // console.log(data);
+    $scope.pins = data.pins;
+    var numberOfPins = data.pins.length;
+    // console.log('from controller I see', angular.element('.box').length,'boxes');
+  });
+
+  $http.get('/categories.json').success(function(data){
+    console.log('sucess on get categories');
+    // console.log(data);
+  $scope.categories = data.categories;
```
Comment (h8rry) [inline app/assets/javascripts/angular/controller/home-controller.js.erb]: Indentation. Try to fix all of them?

**Comment by cassiemak** [comment]
Made all the changes. Please merge. Thanks!

File: app/assets/templates/header.html [hunk 7]
```diff
@@ -1,10 +1,10 @@
...
   <div class="row">
     <div class="header-bar col-xs-10 col-xs-offset-1">
       
       <div class="header-left col-xs-10">
         <div class="row">
-          <a href="#" class="lineup btn btn-default col-xs-1" id="logo"><img src="assets/logo.gif"  height="20px"></a>
+          <a href="#/" class="lineup btn btn-default col-xs-1" id="logo"><img src="assets/logo.gif"  height="20px"></a>
           <input type="text" id="search-bar" class="btn btn-default lineup col-xs-9" placeholder="Search">
           <button id="dropdown" class="lineup btn btn-default dropdown-toggle col-xs-1" data-toggle="dropdown">
             <div class="glyphicon glyphicon-list"></div>
```
Comment (h8rry) [inline app/assets/templates/header.html:10]: `<a href="#">{{ category.name }}</a>` 

Are you ready to add the actual href?

Comment (cassiemak) [inline app/assets/templates/header.html:10]: Yes wor... 
that's the logo that goes back to the root file, "Home" page...
local host works wor...

**Pull request merged**